lib/propolis/Cargo.toml

```diff
@@ -33,6 +33,9 @@ once_cell.workspace = true
 crucible-client-types = { workspace = true, optional = true }
 crucible = { workspace = true, optional = true }
 oximeter = { workspace = true, optional = true }
+nexus-client.workspace = true
+internal-dns.workspace = true
+omicron-common.workspace = true
```

Comment:
I didn't see references to `internal_dns` or `omicron_common` added to propolis-lib.  Did I miss them, or can these deps be elided?

Reply:
They have been elided.
There has to be some rust tool that will do this for you, right?

Reply:
Looks like `cargo-udeps` is the ticket for that, but I'm also not certain about how well it handles cases like workspace dependencies.